python/chronos/src/setup.py

```diff
@@ -53,16 +53,17 @@ def setup_package():
         license='Apache License, Version 2.0',
         url='https://github.com/intel-analytics/BigDL',
         packages=get_bigdl_packages(),
-        install_requires=['bigdl-orca=='+VERSION, 'bigdl-nano[pytorch]',
-                          'pandas>=1.0.5, <1.3.0', 'scikit-learn', 'statsmodels',
-                          'onnx', 'onnxruntime'],
+        # neural_compressor, prophet, openvino-dev
+        install_requires=['bigdl-orca=='+VERSION, 'bigdl-nano=='+VERSION,
+                          'pandas>=1.0.5, <1.3.0', 'scikit-learn>=0.22.0, <=1.0.2', 'statsmodels',
+                          'onnx>=1.8.0, <=1.11.0', 'onnxruntime>=1.6.0, <=1.11.1', 'pmdarima~=1.8.4',
+                          'tsfresh==0.17.0', 'pyarrow~=6.0.1'],
```

Comment:
remove 'pmdarima~=1.8.4', 'tsfresh==0.17.0', 'pyarrow~=6.0.1' for now

Reply:
fixed